packages/twenty-server/src/modules/webhook/standard-objects/webhook.workspace-entity.ts

```diff
@@ -60,4 +60,15 @@ export class WebhookWorkspaceEntity extends BaseWorkspaceEntity {
   })
   @WorkspaceIsNullable()
   description: string;
+
+  @WorkspaceField({
+    standardId: WEBHOOK_STANDARD_FIELD_IDS.secret,
+    type: FieldMetadataType.TEXT,
+    label: 'Secret',
+    description:
+      'Optional secret used to compute the HMAC signature for webhook payloads. This secret is shared between Twenty and the webhook consumer to authenticate webhook requests.',
+    icon: 'IconLock',
+  })
+  @WorkspaceIsNullable()
```

Comment:
I was trying to follow the pattern with `description`. Does this mean we should remove @WorkspaceIsNullable from `description` as well?

Reply:
Arf I didn't see that, good catch. To remove workspace description we should create an upgrade-command otherwise if there are already null items in the DB then the migration isn't going to run. So either create an upgrade version command for 0.40(there are examples in the codebase already). So either do the command or just leave the description as it is and there will be an inconsistency.

Reply:
Makes sense - I'll create another GitHub issue to handle the workspace description cleanup separately, as it's out of scope for this PR.